runtests.sh

```diff
@@ -77,7 +77,7 @@ run_integration(){
         export SNAPCRAFT=snapcraft-coverage
     fi
 
-    python3 -m unittest discover -b -v -s integration_tests
+    SNAPCRAFT_FROM_SOURCE=1 python3 -m unittest discover -b -v -s integration_tests
```

Comment:
> What we could do is to add an arg to ./runtests integration so you can pass snapcraft.tests.foo. So my idea is that you would use ./runtests if you want to check the source, and adt-run if you want to check the installed.

Yeah I do this often enough I'd have to add that variable to my .bashrc. It'd be better if runtests accepted the test module, or we did `SNAPCRAFT_FROM_INSTALLED` instead (since that's only done on CI).